src/Components/Filtering/Filtering.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable max-len */
+import React, { useState } from 'react';
+import Button from '@mui/material/Button';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import propTypes from 'prop-types';
+import RangeFilter from './RangeFilter';
+import MultSelectElem from './Multiselect';
+
+const gradeRangeMetadata = ['0 to Pre-K', '1st', '2nd', '3rd', '4th', '5th', '6th', '7th', '8th', '9th', '10th', '11th', '12th'];
+const ageRangeMetadata = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18'];
+
+export default function Filter({ setRangeState, setMultiSelectInput }) {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [rangeFilterData, setRangeFilterData] = useState({ age: { min: ageRangeMetadata[0], max: ageRangeMetadata[18] }, grade: { min: gradeRangeMetadata[0], max: gradeRangeMetadata[12] } });
```

Comment:
maybe change this to tempRangeFilter if it's going to be used the same way as the multiselect